src/main/java/com/ethercamp/harmony/web/controller/MainController.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015, 2016 Ether.Camp Inc. (US)
+ * This file is part of Ethereum Harmony.
+ *
+ * Ethereum Harmony is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Ethereum Harmony is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Ethereum Harmony.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.ethercamp.harmony.web.controller;
+
+import com.ethercamp.harmony.config.RpcEnabledCondition;
+import com.ethercamp.harmony.util.AppConst;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class MainController {
+
+    /**
+     * Handles JSON-RPC requests at site root
+     */
+    @Conditional(RpcEnabledCondition.class)
+    @RequestMapping(value = {AppConst.JSON_RPC_ALIAS_PATH}, method = RequestMethod.POST)
+    public String jsonrpcAlias() {
+        return "forward:" + AppConst.JSON_RPC_PATH;
```

Comment:
Won't that just forward everything sent to root to the `/rpc` endpoint? If yes, then what is the case for that?